messaging/kafka-avro-reactive-messaging/pom.xml

```diff
@@ -10,6 +10,11 @@
     <artifactId>kafka-avro-reactive-messaging</artifactId>
     <name>Quarkus OpenShift TS: Messaging: Reactive Kafka with Avro</name>
     <dependencies>
+        <!-- TODO: com.squareup.retrofit2:retrofit:2.9.0 should be removed on Quarkus 2.0.0.Alphas-->
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+        </dependency>
```

Comment:
We don't need to explicitly add this dependency here. Adding it into the dependency management is enough.